### PR TITLE
docs: source docs added for moja.modules.cbm

### PR DESCRIPTION
**Pull request by Namyalg**

### CODE REVIEW

**Comment by aornugent** [comment]
Thanks @Namyalg - can you please share the link to your example? Last time I checked it was looking fantastic.

**Comment by Namyalg** [comment]
Deployed link : https://namyalg.github.io/moja.canada/
It is built into the CI

**Comment by aornugent** [comment]
@HarshCasper - are the CI errors permission related? Happy to merge if so.

**Review by aornugent — APPROVED** [review]

File: Source/moja.modules.cbm/include/moja/modules/cbm/record.h [hunk 1]
```diff
@@ -262,7 +262,7 @@ namespace cbm {
         PoolRecord(Int64 locationId, Int64 poolId, double value);
         ~PoolRecord() {}
 
-        bool operator==(const PoolRecord& other) const;
+        bool operator==(const PoolRecorother) const;
```
Comment (aornugent) [inline Source/moja.modules.cbm/include/moja/modules/cbm/record.h]: This looks like a typo

File: Source/moja.modules.cbm/include/moja/modules/cbm/rootbiomassequation.h [hunk 2]
```diff
@@ -47,7 +47,7 @@ namespace cbm {
 
         virtual ~SoftwoodRootBiomassEquation() = default;
 
-        inline double calculateRootBiomass(double componentAbovegroundCarbon) {
+        inline double (double componentAbovegroundCarbon) {
```
Comment (aornugent) [inline Source/moja.modules.cbm/include/moja/modules/cbm/rootbiomassequation.h]: This deletion doesn't look right.

**Review by aornugent — CHANGES_REQUESTED** [review]
@HarshCasper @radistoubalidis - can we build this branch and run the FLINTcloud integration test to check that the only documentation (and not GCBM source) has been modified.

**Review by aornugent — APPROVED** [review]

**Comment by Namyalg** [comment]
I have gone through the changes, it will be a good idea to run a test and ensure the source code has not changed.

**Review by aornugent — APPROVED** [review]
@Namyalg has checked that her changes affect docs only. @mfellows - please let us know if you encounter any issues and so that we can rollback and debug.

Thanks @Namyalg for this heroic effort!

**Comment by mfellows** [comment]
Thanks @Namyalg, that's a huge amount of work!

**Pull request merged**